controllers/petitions.controller.js

```diff
@@ -166,7 +180,10 @@ petitionsController.createPetitionWithItems = catchAsync(
 // - Allows a client to retrieve a list of petitions from the use.
 // - Often produces related data. The comments of a post for a example.
 petitionsController.read = catchAsync(async (req, res) => {
-  const petitions = await Petition.find({}).populate("owner").populate("items");
+  const petitions = await Petition.find({})
```

Comment:
On my local this endpoint doesn't populate the owner field. Does it on yours? Somewhere this handler broke

![Screen Shot 2021-08-15 at 1 17 34 PM](https://user-images.githubusercontent.com/16382165/129469134-a1ab5c76-806b-4352-a87e-4b64e5e96535.png)

Reply:
@cqnguy23 After reviewing the commit history I see this is the one where we update `.read` handler.

https://github.com/cqnguy23/Hackathon_Project/commit/a3dd2dfdb46f053eca3a970d6434fbce30a65f3d

Could you please let me know if `owner` populates on your machine? I've been playing around with it and haven't been able to.

Reply:
One more thing, in that commit we can see we went from `petitions` to `newPetitions` inside of`petitions.controller.js`.

Thus our frontend broke and we needed to update to parse the response correctly [here](https://github.com/PrimeTimeTran/Hackathon-fe/pull/43). Could you please review and approve it looks good?

Reply:
It might have been [this commit](https://github.com/cqnguy23/Hackathon_Project/commit/d9bf376c84746f769fc8075d2118857b9d23a7f6) as well that broke the population of `owner`.

Reply:
Please check this
[lastest commit] (https://github.com/cqnguy23/Hackathon_Project/pull/24/commits/d08191b31ebb2d78c82dab327eda8d194a42d837)